crates/sovereign-sdk/rollup-interface/src/state_machine/da.rs

```diff
@@ -223,6 +223,9 @@ pub trait BlockHeaderTrait: PartialEq + Debug + Clone + Serialize + DeserializeO
     /// Hash the type to get the digest.
     fn hash(&self) -> Self::Hash;
 
+    /// Merkle root of the block.
+    fn merkle_root(&self) -> Self::Hash;
```

Comment:
let's call this transaction_commitment everywhere, some da layers might not use merkle roots